log/options.go

```diff
@@ -159,6 +164,13 @@ func (o *Options) WithTeeToStackdriver(project, logName string, mr *monitoredres
 	return o
 }
 
+func (o *Options) WithTeeToUDS(addr, path string) *Options {
```

Comment:
nit: can we add a comment here about the protocol and the use cases, so that anyone that stumbles on this option can understand more of what this is about?

Reply:
Done.